packages/file-search/src/browser/quick-file-open.ts

```diff
@@ -69,10 +78,12 @@ export class QuickFileOpenService implements QuickOpenModel, QuickOpenHandler {
      */
     protected isOpen: boolean = false;
 
+    protected filterAndRangeQueryDefault = { filter: '', range: { start: { line: 0, character: 0 }, end: { line: 0, character: 0 } } };
```

Comment:
We can remove the use of `query`:

```suggestion
    protected filterAndRangeDefault = { filter: '', range: { start: { line: 0, character: 0 }, end: { line: 0, character: 0 } } };
```

Reply:
Done, I agree with this renaming as well !!